checkov/common/sca/output.py

```diff
@@ -422,37 +462,40 @@ def add_cve_record_to_report(vulnerability_details: dict[str, Any], package_name
     report.add_record(cve_record)
 
 
-def apply_inline_suppressions(
-        record: Record, vulnerability_details: dict[str, Any], inline_suppressions: _ScaSuppressions | None = None
+def apply_cves_inline_suppressions(
+        record: Record, inline_suppressions_maps: _ScaSuppressionsMaps | None = None
 ) -> bool:
     """Applies the inline suppression and returns an accomplish status"""
 
-    if inline_suppressions:
-        if "package" in inline_suppressions:
-            package_suppression = inline_suppressions["package"].get(vulnerability_details.get("packageName", ""))
-            if package_suppression:
-                if "suppress_comment" in package_suppression:
-                    record.check_result = {
-                        "result": CheckResult.SKIPPED,
-                        "suppress_comment": package_suppression["suppress_comment"],  # type:ignore[typeddict-item]
-                    }
-                    return True
-                else:
-                    # CVE suppression for specific package
-                    cve_suppression = package_suppression.get(vulnerability_details.get("cveId", ""))
-                    if cve_suppression:
-                        record.check_result = {
-                            "result": CheckResult.SKIPPED,
-                            "suppress_comment": cve_suppression["suppress_comment"],  # type:ignore[index]
-                        }
-                        return True
-        if "cve" in inline_suppressions:
-            cve_suppression = inline_suppressions["cve"].get(
-                vulnerability_details.get("cveId", vulnerability_details.get("id", "")))
+    if inline_suppressions_maps:
+        if inline_suppressions_maps.get("cve_by_cve_map") and record.vulnerability_details:
```

Comment:
```suggestion
        if record.vulnerability_details and inline_suppressions_maps.get("cve_by_cve_map"):
```
it is faster to check if a value is set compared to a dict lookup